packages/common/src/system/guid-utils.ts

```diff
@@ -8,21 +11,22 @@ import * as uuid from 'uuid-with-v6';
 /**
  * UUID RFC 4122 https://tools.ietf.org/html/rfc4122
  */
-export namespace Guid {
+@injectable()
+export class GuidUtils {
     /**
      * Creates new UUID version 6.
      */
-    export function createGuid(): string {
+    public createGuid(): string {
         return uuid.v6();
     }
 
     /**
      * Creates UUID with the same UUID node part (the last 12 bytes) as base UUID.
      * @param baseGuid The UUID to match the UUID node with.
      */
-    export function createGuidForNode(baseGuid: string): string {
+    public createGuidForNode(baseGuid: string): string {
```

Comment:
the name is bit confusing, it seems like we are creating Guid for Node. Instead you probably can change it to createNewGuidFromBaseGuid(); generateGuidFromBaseGuid() etc. And it will be clear to the caller what the function is going to do.